.storybook/main.ts

```diff
@@ -16,6 +17,16 @@ const config: StorybookConfig = {
     getAbsolutePath('@storybook/addon-essentials'),
     getAbsolutePath('@storybook/addon-interactions'),
     getAbsolutePath('@storybook/addon-links'),
+    {
```

Comment:
This is required so we can use advanced Markdown syntax in the READMEs such as tables.